### PR TITLE
feat: add integration with query in form

### LINKED ISSUE #48: Add Check view Page
### Explication
 When this is mounted, a modal with the message: 'Antes de recibir..' appears, this modal is being worked in other Issue (Check the first reference).

 This issue is about what occurs after the modal is closed

### Screenshots
 | Mobile | Desktop |
 | --- | --- |
 | image |     image |

### Checklist
 This issue can be closed when the following tasks are complete:

- [x] #41 
- [x] When user enter in the view the modal with the message: 'Antes de recibir...' appears
- [x] When user clicks 'Entendido' modal button, the modal is closed
- [x] Layout elements (let's call it BLUE and BG) changes a lot. BLUE loses the border-radius and BG changes color! Also, the title of the form changes color
- [ ] Image changes align from start to end in her div
- [ ] Back button gains a Text (why not have it across all sizes?
- [ ] This should show input errors (ticket doesn't exist on DB, format not allowed, others TBD)
- [ ] This should show error toast for server errors (server unavailable, server error), others TBD), or Client High Severity errors.
- [ ] Clicking Button should redirects to Map view with a ticket as param

### Questions
 - [ ] What happens when the Ticket state changes to `destination_reached` or `delivery_confirmed`? Changes the view/show modal?
 - [ ] Should Button be disabled if no Ticket is given?
 - [ ] Should the button have a `in_travel` state?

### References
 - Mobile: https://xd.adobe.com/view/c94ee522-180f-487f-b7e0-90754a4b6169-9c1d/screen/70fa64b6-de96-4c98-b934-77504dc10b22
 - Desktop: https://xd.adobe.com/view/b9a29d6f-091e-47d3-8769-14f54102d9e1-ecf1/screen/5e636c6c-6454-4de0-9312-55c1d326619f/
 
#### Related issues
- #41  
- #20
- #24
- #25
- #26
- #32

### PR DESCRIPTION
**Pull request by denisosuna**
### Description

Was added validation for the input, in order to check code tipped with DB

fixes #48 

#### Screenshots

If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

1. Go to '/check'
2. fill the input with test and it will redirect otherwise it will throw an alert

### Checklist:

- [x] Doesn't break the current code.
- [x] Passes linters and test, also is building.
- [x] Doesn't have spelling or grammatical problems.
- [x] Doesn't have unnecessary comments or debugging code.
- [x] The branch is updated with the last dev branch changes.

### CODE REVIEW

File: src/pages/Check.jsx [hunk 1]
```diff
@@ -6,15 +6,23 @@ import { Button, Input, Modal, Text } from '../components'
...
+  const toDelivery = async e => {
     e.preventDefault()
-    history.push('/delivery/test')
+    const { packages } = await packageInformation
+    if ((await packages.length) > 0) {
+      history.push(`/delivery/${packageCode}`)
+    }
+
+    if ((await packages.length) === 0) {
+      alert('el packcode no existe')
```
Comment (Israel-Laguan) [inline src/pages/Check.jsx]: use log Error

File: src/pages/Check.jsx [hunk 2]
```diff
@@ -6,15 +6,23 @@ import { Button, Input, Modal, Text } from '../components'
...
+    }
```
Comment (Israel-Laguan) [inline src/pages/Check.jsx]: What about
```suggestion
    if ((await packages.length) > 0) {
      history.push(`/delivery/${packageCode}`)
    }

    alert('el packcode no existe')
```

The idea is that the IF only can have 2 results, and your second conditional is precisely the other result, so is not really needed to create another IF block

**Review by Israel-Laguan — COMMENTED** [review]
Please check the comments

File: src/pages/Check.jsx (hunk 1 above)
Comment (denisosuna) [reply src/pages/Check.jsx]: but this is not an error, it should throw an error message that the id does not exist or something

Comment (Israel-Laguan) [reply src/pages/Check.jsx]: Got it, maybe in the future we can implement a toast for this

For the moment being, is OK like this

**Pull request merged**